src/main/java/team/returm/jobis/domain/company/domain/repository/CompanyRepository.java

```diff
@@ -23,16 +30,54 @@ public List<StudentQueryCompaniesVO> findCompanyInfoList() {
         return queryFactory
                 .select(
                         new QStudentQueryCompaniesVO(
-                             QCompany.company.name,
-                             QCompany.company.companyLogoUrl,
-                             QCompany.company.sales
+                             company.name,
+                             company.companyLogoUrl,
+                             company.sales
                         )
                 )
-                .from(QCompany.company)
-                .orderBy(QCompany.company.name.desc())
+                .from(company)
+                .orderBy(company.name.desc())
                 .fetch();
     }
 
+    public QueryCompanyDetailsResponse queryCompanyDetails(UUID companyId) {
+        return queryFactory
+                .select(
+                        new QQueryCompanyDetailsResponse(
+                                company.bizNo,
+                                company.companyLogoUrl,
+                                company.companyIntroduce,
+                                company.address.mainZipCode,
+                                company.address.mainAddress,
+                                company.address.subZipCode,
+                                company.address.subAddress,
+                                company.manager.managerName,
+                                company.manager.managerPhoneNo,
+                                company.manager.subManagerName,
+                                company.manager.subManagerPhoneNo,
+                                company.fax,
+                                company.email,
+                                company.representative,
+                                company.foundedAt,
+                                company.workersCount,
+                                company.sales,
+                                recruitment.id
+                        )
+                )
+                .from(company)
+                .leftJoin(company.recruitmentList, recruitment)
+                .where(
+                        company.id.eq(companyId),
+                        recruitment.recruitYear.eq(Year.now().getValue()),
+                        recruitment.status.eq(RecruitStatus.RECRUITING),
+                        recruitment.eq(
+                                selectFrom(recruitment)
+                                        .orderBy(recruitment.createdAt.desc())
```

Comment:
```suggestion
                .leftJoin(recruitment)
                .on(
                        recruitment.company.id.eq(company.id),
                        recruitment.createdAt.eq(
                                JPAExpressions.select(recruitment.createdAt.max())
                                        .from(recruitment)
                                        .where(
                                                recruitment.company.id.eq(company.id),
                                                recruitment.status.eq(RecruitStatus.RECRUITING)
                                        )
                        )
                )
                .where(company.id.eq(companyId))
```
여기 쿼리가 이러면 의미가 없을것 같아서 이런식으로 하면 좋을것 같아요